### ISSUE TITLE
Error connecting to SQL Server using WhiteRabbit

### ISSUE THREAD

**Issue by solmazeradat-aridhia** [OHDSI/WhiteRabbit#344]
Hi,

I am trying to test the connection to the Microsoft SQL server where we have source data ready to be scanned by WhiteRabbit. After populating the **source data location**, the error below is:

Exception in thread “AWT-EventQueue-0” java.lang.RuntimeException: Cannot connect to DB server: The server principal “user” is not able to access the database “master” under the current security context. ClientConnectionId:0ab964c4-8267-40c5-b105-32334516edb7

![image](https://user-images.githubusercontent.com/91956839/182891512-5905b373-5efa-41e8-b884-307b7cfc5b6c.png)

Also raised this in the forum - https://forums.ohdsi.org/t/error-connecting-to-sql-server-using-whiterabbit/17109

Many thanks,
Solmaz

**Comment by solmazeradat-aridhia** [comment]
Hi @MaximMoinat,

Managed to fix the issue when connecting our Microsoft SQL server to White Rabbit. Since we are using Azure as a hosting service, the option we needed to select was "Azure" (More detailed points on the troubleshooting can be seen in the forum question I posted). 

 From a user perspective it was not clear at first "Azure" was the option which needed to be selected. Initially, we selected "SQL server" because this is server type we are using. It was after reading through the [documentation ](http://ohdsi.github.io/WhiteRabbit/WhiteRabbit.html#running-from-the-command-line) that we realized that "Azure SQL Database" was the same as "Azure" from the dropdown menu in White Rabbit. 

Aligning the wording between the Dropdown option and the documentation might be worth considering. 

Many thanks,
Solmaz
